### DISCUSSION TITLE
submission format

### DISCUSSION THREAD

**Discussion by nikkiwoo-gh** [lucaro/DRES#260] [Q&A] [answered]
Hi there,

I follow the submission guideline "http(s)://{server}/submit?item={item}?frame={frame}" to submit the result for KIS.

When I try this: http://ip:8080/submit?session=node01axjtosxap2mdmy4i0elhpel80&item=3439&frame=7823, it responses:
404: {"description":"Media item '3439 (collection = UID(2099773e-e0c2-486e-85d0-7b694f44629f))' could not be found.","status":false}
However, the correct result is the video media having the name "03439" in the collection.

I also try: http://ip:8080/submit?session=node01axjtosxap2mdmy4i0elhpel80&item=03439&frame=7823 (where item==media item name), still gets
400: {"description":"Item '03439' not found.","status":false}

Could you please provide some hints to solve this problem?
I wonder what is the format for "item", and when it performs matching to the answer, is it match to the video name showing in the collection?
Thanks a lot

**Comment by lucaro** [comment, marked as answer]
Hey,

is this happening on an instance of DRES which you setup yourself? If so, which version are you using? What are you using as a media collection?

**Reply by nikkiwoo-gh** [reply]
Hi,

Yes. I am using the DRES setup by myself. I was using release 0.7. I also try the latest version 0.7.1a, and the same error happens.

I am using V3C1, and each video follows this name format "%05d.mp4", e.g., 00001.mp4. All the videos can be played in the frontend (media collection management). "00001.mp4" is with name "00001", relative location "00001.mp4", type "video"... The video also can be previewed when I set up the competition task.

**Reply by lucaro** [reply]
I see, this might be due to an issue we fixed in the latest release. Try updating to 0.8 to see if it resolves the issue.

**Reply by nikkiwoo-gh** [reply]
It is working fine now!! Thanks a lot
